.gitignore

```diff
@@ -7,6 +7,8 @@
 /testing_baseline
 /testing_upgrades
 /resources/buildstock.csv
+/project_*/**/*.csv
```

Comment:
Why are you ignoring .csv? I think this is causing the source_report.csv and options_saturations.csv to be deleted.

Reply:
yeah, @afontani wanted only the tsvs to be in projects folder.